src/client/core/utils/dom.js

```diff
@@ -84,9 +87,9 @@ function sortElementsByFocusingIndex (elements) {
 }
 
 function insertIFramesContentElements (elements, iFrames) {
-    const sortedIFrames         = sortElementsByTabIndex(iFrames);
+    const sortedIFrames       = sortElementsByTabIndex(iFrames);
     let results               = [];
-    const iFramesElements       = [];
+    const iFramesElements     = [];
```

Comment:
please rename this var into `iframeElement` because we agreed to use this style

Reply:
do you mean just to use 'f' in lower case?

Reply:
yes